crates/lib3h/src/lib.rs

```diff
@@ -39,6 +39,17 @@ pub mod transport;
 // FIXME
 // pub mod transport_wss;
 
+// Global Tracer
+lazy_static! {
+    pub static ref LIB3H_TRACER: std::sync::Mutex<holochain_tracing::Tracer> =
+        std::sync::Mutex::new(holochain_tracing::null_tracer());
+}
+
+pub fn new_root_span(op_name: &str) -> holochain_tracing::Span {
+    let tracer = LIB3H_TRACER.lock().unwrap();
+    tracer.span(format!("(root) {}", op_name)).start().into()
+}
+
```

Comment:
do you think it would make more sense for these two functions to be in the holochain_tracing library, so lib3h can set it to a test tracer for unit / integration tests, but holochain core can set it to whatever it wants when we start expanding tracing?

Reply:
Depends if you want one global tracer for everything holochain or have a separate one for lib3h.
I'm not sure whats the best pattern here, thats why I have both LIB3H_TRACER and GHOST_TRACER in the ghost crate.
Maybe this global could hold a reference instead?